docs/providers/aws/cli-reference/install-plugin.md

```diff
@@ -0,0 +1,36 @@
+<!--
+title: Serverless Framework Commands - AWS Lambda - Install Plugin
+menuText: Install Plugin
+menuOrder: 4
+description: Install a Serverless plugin
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/aws/cli-reference/install-plugin)
+<!-- DOCS-SITE-LINK:END -->
+
+# Install Plugin
+
+Install a Serverless plugin and add it to the services `plugins` array.
+
+**Note:** You might want to change the order of the plugin in the services `plugins` array.
```

Comment:
lol why?!

Reply:
Because plugins are loaded in the order you list them in the `plugins` array.

I you e.g. want to overwrite a functionality from another plugin you'd have to put this plugin after the one you want to overwrite.